README.md

```diff
@@ -258,6 +208,106 @@ const clientOpts = await connector.getOptions({
 });
 ```
 
+### Automatic IAM Database Authentication
+
+Connections using [Automatic IAM database authentication][] are supported when
+using Postgres or MySQL drivers.
+
+Make sure to [configure your Cloud SQL Instance to allow IAM authentication][configure-iam-authn]
+and [add an IAM database user][add-iam-user].
+
+A `Connector` can be configured to connect to a Cloud SQL instance using
+automatic IAM database authentication with `getOptions` through the
+`authType` argument.
+
+```js
+const clientOpts = await connector.getOptions({
+  instanceConnectionName: 'my-project:region:my-instance',
+  authType: 'IAM'
+});
+```
+
+When configuring a connection for IAM authentication, the `password` argument
+can be omitted and the `user` argument should be formatted as follows:
+> Postgres: For an IAM user account, this is the user's email address.
+> For a service account, it is the service account's email without the
+> `.gserviceaccount.com` domain suffix.
+>
+> MySQL: For an IAM user account, this is the user's email address, without
+> the `@` or domain name. For example, for `test-user@gmail.com`, set the
+> `user` field to `test-user`. For a service account, this is the service
+> account's email address without the `@project-id.iam.gserviceaccount.com`
+> suffix.
+
+Examples using the `test-sa@test-project.iam.gserviceaccount.com`
+service account to connect can be found below.
+
```

Comment:
I wonder if we should add a note here mentioning that SQL Server is not supported 🤔 something like:

```
Note: SQL Server IAM authentication is not supported at this point.
```

Reply:
let's go with just the first sentence that mentions it's only supported on Postgres and MySQL instead 👍